src/screens/landingPage.js

```diff
@@ -1,9 +1,142 @@
 import React from 'react'
-
 import {
-  BodyText,
-} from '../components'
+  StyleSheet,
+  Animated,
+  View,
+  Image,
+  Text,
+  TouchableOpacity,
+} from 'react-native'
+
+var styles = StyleSheet.create({
+  container:{
+    flex: 1,
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    backgroundColor: '#0D3B66',
+    paddingTop: 80,
+    paddingBottom: 80,
+    paddingLeft: 25,
+    paddingRight: 25,
+  },
+  title: {
```

Comment:
This is like all good, it just makes me sad that all the file structure i made is just out the window :(... It's fine, but can we move the font stuff into the global styles object? merp, I'll just catch u in the morning.